### ISSUE TITLE
[MHV Portal / Accessibility] My HealtheVet current section indicator does not have sufficient contrast

### ISSUE THREAD

**Issue by sarahhortonadhoc** [department-of-veterans-affairs/va.gov-team#128661]
### 📝 User story

As a user, I want the element indicating the current section of the site to be visible and distinguishable, so that I know where I am in the site.

### 📝 Problem statement

The current section indicator lacks contrast with its background.

### 🌌 Supporting screenshots or videos

<img width="1013" height="182" alt="Screenshot of website section, subsection, and breadcrumb navigation showing low-contrast current section indicator" src="https://github.com/user-attachments/assets/c33d6c5e-0eb6-4ce2-b01c-fe161695833f" />

### 👣 Reproduction steps

1. Navigate to My HealtheVet on va.gov.
2. Notice the blue underline beneath the My HealtheVet link in the global navigation.

### 🔗 Point of contact

@sarahhortonadhoc

### ⚠️ Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

#### Accessibility standards
- [Guideline 1.4 Distinguishable: Make it easier for users to see and hear content including separating foreground from background.](https://www.w3.org/WAI/WCAG22/Understanding/distinguishable)
 - [SC 1.4.11 Non-text Contrast (Level AA)](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast)

#### Experience Standards
- [Findability: User can identify where they are within VA.gov or VA Mobile App and where to go next to complete a task](https://design.va.gov/about/experience-standards/#category-findability)
  - [User can determine where a page or screen is within VA.gov or VA Mobile App](https://design.va.gov/about/experience-standards/#user-can-determine-where-a-page-or-screen-is-within-vagov-or-va-mobile-app)

### 𝌣 Proposed solution or next steps

Design and implement a current section indicator that has a contrast ratio of at least 3:1 against adjacent colors.

### ✅ Acceptance criteria

- [ ] The system displays a current section indicator that has sufficient contrast against the background color.
- [ ] The user can see the current section indicator when they are in the My HealtheVet section.

### ✅ Possible tasks

- [ ] Review, refine, and acknowledge feedback.
- [ ] Update ticket information (title, assignee, labels, project(s), etc.)
- [ ] Fix and/or document decisions made.
- [ ] Product manager to move ticket in their respective product board to 'QA/A11y Review' after reviewing documented decisions / validating fix.
- [ ] Accessibility specialist will close the ticket after reviewing documented decisions / validating the fix.

**Comment by sarahhortonadhoc** [comment]
- DS team closed the ticket with a [comment and rationale for not making a change](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5942#issuecomment-4217219879). 
- MHV Accessibility submitted a [followup comment and rationale for revisiting the decision](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/5942#issuecomment-4224058248).
